### PR TITLE
Env lighting fixes

### PR DESCRIPTION
**Pull request by chornesays**
Updated the prefab structure to remove lights and move it into the simulator. Fixed the skybox to be similar to our env lighting setup.

### CODE REVIEW

File: Core/templateScene.unity [hunk 1]
```diff
@@ -750,6 +906,121 @@ MonoBehaviour:
...
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 2029108019}
+  m_Enabled: 1
+  serializedVersion: 10
+  m_Type: 1
+  m_Shape: 0
+  m_Color: {r: 1, g: 1, b: 1, a: 1}
+  m_Intensity: 1
```
Comment (gazuntype) [inline Core/templateScene.unity]: i think we may have to reduce the intensity of the default sunlight. when i created a filter and loaded it on the app, the app version was significantly darker than the in-plugin version even though my room is pretty bright. so i'm thinking we reduce the intensity of the sunlight so as to not give a wrong impression of how bright environment light actually gets.

**Review by gazuntype — CHANGES_REQUESTED** [review]
good stuff! just a suggestion and a requested change!

File: Core/templateScene.unity (hunk 1 above)
Comment (chornesays) [reply Core/templateScene.unity]: ooh. this is a mistake. i had swapped to .7 but it fell off. good catch

File: package.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
 {
   "name": "com.getfilta.artist-unityplug",
   "displayName": "Filta Artist Suite",
-  "version": "0.1.1",
-  "unity": "2020.1",
+  "version": "0.4.0",
```
[the remark this answers is not in the document]
Comment (chornesays) [reply package.json]: Done

**Pull request merged**